Sources/JS/printCollectedTypes.js

```diff
@@ -0,0 +1,77 @@
+//https://github.com/beatgammit/base64-js/blob/master/index.js
```

Comment:
Is the code below copied from here? We probably can't do that due to license issues... Depending on the license, we might be able to include this code with its original license as comment, but I'm not 100% sure about that. Alternatively, maybe it's possible to avoid the need for this code by using the JSON encoding (see main comment)?

Reply:
it is under MIT license so that should not be a problem, but yeah we can change JS->Swift communication